operate/client/src/App/Processes/MigrationView/BottomPanel/index.test.tsx

```diff
@@ -531,8 +541,8 @@ describe('MigrationView/BottomPanel', () => {
       }),
     ).toBeVisible();
 
-    // Expect all 29 rows to be visible (+1 header row)
-    expect(await screen.findAllByRole('row')).toHaveLength(30);
+    // Expect all 31 rows to be visible (+1 header row)
+    expect(await screen.findAllByRole('row')).toHaveLength(32);
```

Comment:
This is an instance of what I referenced on the other comment about arbitrary values in the code that require comments for context.